qutip/solver/flimesolve.py

```diff
@@ -0,0 +1,666 @@
+                rate_products * time_sense
+            )
+            if not np.any(included_deltas):
+                continue
+            keys = np.unique(delta_shift[included_deltas])
+            for key in keys:
+                mask[key] = np.logical_and(delta_shift == key, included_deltas)
+
+            for key in mask.keys():
+                valid_c_op_products = rate_products * mask[key]
+                I_ = np.eye(Hdim, Hdim)
+                flime_FirstTerm = np.transpose(
+                    valid_c_op_products, [0, 2, 1, 3]
+                ).reshape(Hdim**2, Hdim**2)
+                tmp = np.trace(valid_c_op_products, axis1=0, axis2=2)
+                flime_SecondTerm = np.kron(tmp.T, I_)
+                flime_ThirdTerm = np.kron(I_, tmp)
+
+                total_R_tensor[key] += flime_FirstTerm - (0.5) * (
+                    flime_SecondTerm + flime_ThirdTerm
+                )
+
+    # Turning the Rate Matrix into a super operator
+    dims = [floquet_basis.U(0).dims] * 2
+    total_R_tensor = {
+        key: Qobj(RateMat, dims=dims, copy=False)
+        for key, RateMat in total_R_tensor.items()
+    }
+
+    return total_R_tensor
+
+
+def flimesolve(
+    H,
+    rho0,
+    tlist,
+    T,
+    Nt=None,
+    c_ops=None,
+    e_ops=None,
+    args=None,
+    time_sense=0,
+    options={},
+):
+    """
+    Solve system dynamics using the Floquet-Lindblad Master Equation
+
+    Parameters
+    ----------
+
+    H : :class:`Qobj`,:class:`QobjEvo`,:class:`QobjEvo` compatible format.
+        Periodic system Hamiltonian as :class:`QobjEvo`. List of
+        :class:`Qobj`, :class:`Coefficient` or callable that
+        can be made into :class:`QobjEvo` are also accepted.
+
+    rho0 / psi0 : :class:`qutip.Qobj`
+        Initial density matrix or state vector (ket).
+
+    tlist : *list* / *array*
+        List of times for :math:`t`.
+
+    T : float
+        The period of the time-dependence of the hamiltonian.
+
+    Nt: int
+        The number of points within one period of the Hamiltonian, used for
+        forming the rate matrix. If none is supplied, flimesolve will try to
+        pull Nt from tlist.
+
+    c_ops : list of (:obj:`.QobjEvo`, :obj:`.QobjEvo` compatible format)
+        Single collapse operator, or list of collapse operators
+
+    e_ops : list of :class:`qutip.Qobj` / callback function
+        List of operators for which to evaluate expectation values.
+        The states are reverted to the lab basis before applying the
+
+    args : *dictionary*
+        Dictionary of parameters for time-dependent Hamiltonian
+
+    time_sense : float
+        Value of the secular approximation to use when constructing the rate
+        matrix R(t).Default value of zero uses the fully time-independent/most
+        strict secular approximation, and will utilize the "diag" solver method.
+        Values greater than zero have time dependence, and will subsequently
+        use the "Adams" method for the ODE solver.
```

Comment:
The way it written it seems that the user input of `method` will be ignored in favor of these.
It only the default that is changing.
```suggestion
        strict secular approximation, values greater than zero have time 
        dependence. The default integration method change depending
        on this value, "diag" for `0`, "adams" otherwise.
```